Gemfile

```diff
@@ -23,7 +23,8 @@ gem 'puma', '~> 3.11'
 # Reduces boot times through caching; required in config/boot.rb
 gem 'bootsnap', '>= 1.1.0', require: false
 
-gem 'graphql' 
+gem 'graphql'
+gem 'sqlite3'
```

Comment:
@susanwere could you kindly explain what this sqlite3 is doing here as you're using PostgreSQL database engine

Reply:
Hey @gtsofa so I Postgres had broken on travis so I was trying to test whether SQlite3 would work fine. It did. And it helped in fixing Postgres since I realised Travis is sort of not compatible with `Postgres - 10` so I fixed it by specifying the Postgres version in .travis.yml file `Postgres - 9.6` as an addon.
 
https://github.com/susanwere/Authors_Haven/blob/b1c34f1b9a93669c84924bc7fcdd0cffa7520e1e/.travis.yml

That fixed the error. I will remove the Gem. Thank you for noticing this.